docs/source/config.json

```diff
@@ -22,8 +23,15 @@
       "10. Authenticate your operations": "/tutorial/10-authenticate-your-queries",
       "11. Write your first subscription": "/tutorial/11-subscriptions"
     },
-    "Essentials": {
+
+    "Configuration": {
       "File types": "/essentials/file-types",
+      "Gradle plugin configuration": "/advanced/plugin-configuration",
+      "Batching operations": "/advanced/query-batching",
```

Comment:
Maybe that could be in the "Networking" part? And move "multi modules" & "codegen methods" in Configuration (as they involve a fair amount of Gradle interaction?

Reply:
Sure. I can update that!